src/ts/components/uicontainer.ts

```diff
@@ -139,6 +148,13 @@ export class UIContainer extends Container<UIContainerConfig> {
     player.addEventHandler(player.EVENT.ON_CAST_STARTED, () => {
       showUi(); // Show UI when a Cast session has started (UI will then stay permanently on during the session)
     });
+    this.playerStateChange.subscribe((_, state) => {
+      playerState = state;
+      if (hidingPrevented()) {
+        this.uiHideTimeout.clear();
+        showUi();
+      }
```

Comment:
There is one issue when the user press `replay` (`playerState` change from `finished` to `playing`) the UI will not hide automatically.

Add something like:
```
      } else {
        if (isUiShown) {
          this.uiHideTimeout.start();
        }
      }
```